packages/manager/modules/pci/src/projects/project/notebooks/add/add.controller.js

```diff
@@ -11,6 +24,30 @@ export default class {
     this.messageContainer = 'pci.projects.project.notebooks.add';
     this.loadMessages();
     this.trackNotebooks('configuration', 'page');
+
+    this.notebook = {
```

Comment:
can you move this object initialisation to notebook.class.js file?

Reply:
We can have a notebook object (of a class) here with the default values. The user can change them as he/she proceeds through the configuration steps

Reply:
No, it's a model to manage creation of notebook.
I changed the name to notebookModel